src/Actions/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import svgPanZoom from 'svg-pan-zoom/src/svg-pan-zoom.js'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTimesCircle, faPlus, faMinus, faUndoAlt } from '@fortawesome/free-solid-svg-icons'
+import Legend from '../Legend'
+import Button from '../Button'
+import ActionGroup from './ActionGroup'
+
+interface Props {
+  mapSvg: SVGSVGElement
+  onClearSelection(): void
+  showLegend: boolean
+  ranges: any // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+interface State {
+  isMobile: boolean
+}
+
+const styles = {
+  container: {
+    position: 'absolute',
+    top: 10,
+    left: 10,
+    right: 10,
+    bottom: 10,
+    display: 'flex',
+    alignItems: 'flex-start',
+    justifyContent: 'space-between',
+    opacity: 0.9,
+    pointerEvents: 'none',
+    whiteSpace: 'nowrap'
+  },
+  icon: {
+    display: 'inline-block',
+    fontSize: 'inherit',
+    height: '1.333333333em',
+    overflow: 'visible',
+    verticalAlign: '-0.125em'
+  }
+}
+
+export default class Actions extends React.Component<Props, State> {
+  container?: HTMLDivElement | null
+  mapZoom: any // eslint-disable-line @typescript-eslint/no-explicit-any
+  timer?: number
+
+  constructor (props: Props) {
+    super(props)
+
+    this.mapZoom = svgPanZoom(this.props.mapSvg, {
+      minZoom: 0.8
+    })
+  }
+
+  componentDidMount () {
+    this.timer = window.setInterval(this.updateIsMobile, 200)
```

Comment:
What do you think about adding a window resize event handler instead of polling every 200ms?

Reply:
That wouldn't handle all of the possible ways the element could change dimensions, such as css animations or direct manipulation via js. It's not the most elegant solution, but it's the best we can do without media queries.

Reply:
We could add media queries with [styled-components](https://www.styled-components.com/docs/advanced#media-templates). Although I just did a search and it doesn't look like we're using `styled-components` in Shell, Core, or seatmaps-client... I thought we were.